packages/components/react/src/core/Terminal/index.tsx

```diff
@@ -13,13 +13,28 @@ export interface TerminalRef {
 export interface TerminalProps {
   theme: 'dark' | 'light';
   className?: string;
+  id?: string;
+  role?: ComponentProps<'div'>['role'];
+  'aria-labelledby'?: string;
```

Comment:
Oh I'm starting to think we should forward everything:

```tsx
export interface TerminalProps extends ComponentProps<'div'> {
  theme: 'dark' | 'light';
  readonly?: boolean;
  onTerminalReady?: (terminal: XTerm) => void;
  onTerminalResize?: (cols: number, rows: number) => void;
}

...

export const Terminal = forwardRef<TerminalRef, TerminalProps>(
  ({ theme, readonly = true, onTerminalReady, onTerminalResize, ...divProps }, ref) => {

...

    return <div {...divProps} ref={divRef} />;
```

What do you think?

Reply:
Sure, done!